src/core/interfaces.ts

```diff
@@ -48,8 +48,6 @@ module Plottable {
   export type _SpaceRequest = {
     width: number;
     height: number;
-    wantsWidth: boolean;
-    wantsHeight: boolean;
```

Comment:
No it doesn't. I was asking for suggestions before I pulled the trigger on refactoring the names.